src/ServiceControl.Monitoring/QueueLength/QueueLengthDataStore.cs

```diff
@@ -2,13 +2,11 @@ namespace ServiceControl.Monitoring.QueueLength
 {
     using System.Collections.Generic;
     using System.Linq;
-    using Http;
     using Newtonsoft.Json.Linq;
-    using Raw;
 
-    class QueueLengthDataConsumer : IRawDataConsumer, IEndpointDataProvider
+    public class QueueLengthDataStore
```

Comment:
I agree that the naming is far from ideal. I would propose to wait with name changes until we do https://github.com/Particular/LaunchWave.DevOpsAdvancedMonitoring/issues/109 which has just started.